experimental/generation/generator/packages/library/templates/standard/en-us/standard.en-us.lg

```diff
@@ -130,8 +130,13 @@ ${'#'} ${type}Entity_Value(entity)
 - ${unexpectedAddDefault()}
 
 >> Units
-
 # unitsTemplates(type)
+- IF: ${propertySchema.minimum || propertySchema.maximum}
+- ${unitsTemplatesWithoutMinMax(type)} ${validationTemplatesWithMinMax()}
```

Comment:
You can simplify to just this, right?  The underlying validations will be empty if there is no min/max.